docs/docs.go

```diff
@@ -12,7 +12,335 @@ import (
 	"github.com/swaggo/swag"
 )
 
-var doc = ``
+var doc = `{
```

Comment:
Assuming this isn't supposed to be here?

Reply:
Good catch, not sure how that happened. I'll take that out.

Reply:
I updated the swag docs using `swag init` so should be a bit nicer now.